src/mlpack/tests/main_tests/logistic_regression_test.cpp

```diff
@@ -438,14 +458,15 @@ BOOST_AUTO_TEST_CASE(LRMaxIterationsChangeTest)
   // Check that the parameters (parameters1 and parameters2) are not equal
   // which ensures Max Iteration changes the output model.
   // arma::all function checks that each element of the vector is equal to zero.
-  BOOST_REQUIRE_MESSAGE(!arma::all((parameters1-parameters2) == 0),
-                        "Parameter(Max Iteration) has no effect on the output");
+  if(!arma::all((parameters1-parameters2) == 0))
+    INFO("Parameter(Max Iteration) has no effect on the output");
```

Comment:
```suggestion
  INFO("Parameter(Max Iteration) has no effect on the output");
  REQUIRE(!arma::all((parameters1-parameters2) == 0))
```

Reply:
@zoq your take on this ?

Reply:
Hey @jeffin143 so how the `BOOST_REQUIRE_MESSAGE` works is it gives the error message that you specified as the second argument but only when it fails and in your change it just shows that as the info no matter if that test fails or not, i have proposed a solution for this in the last comment in the PR, what do you think about that ?